.changeset/fuzzy-llamas-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': minor
```

Comment:
Is a changed file path for an asset a breaking change? I _guess_ so, but since we weren't using it yet, and since it wasn't a code change, I chose `minor`. Second-guessing that now.

Reply:
I'm fine either way :+1: